README.md

```diff
@@ -1,11 +1,64 @@
-# ThinkSmall
-## Project Description
+<p align="center">
+  <a href="https://github.com/othneildrew/Best-README-Template">
+    <img src="screenshots/logo.png" alt="Logo" width="80" height="80">
+  </a>
+
+  <h3 align="center">ThinkSmall</h3>
+
+  <p align="center">
+    A platform to connect small business owners to local volunteers!
+    <br />
+    <a href="https://github.com/CPSC436/ThinkSmall"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/CPSC436/ThinkSmall">View Demo</a>
+    ·
+    <a href="https://github.com/CPSC436/ThinkSmall/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/CPSC436/ThinkSmall/issues">Request Feature</a>
+  </p>
+</p>
+
+## Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Built with](#built-with)
+* [Getting Started](#getting-started)
+* [Project Requirements](#project-requirements)
+* [Minimal Requirement Break Down](#minimal-requirement-break-down)
+* [Prototypes](#prototypes)
+* [User Workflow](#user-workflow)
+* [Troubleshooting](#troubleshooting)
+
+## About the Project
 
 This is a web application that will help small businesses in Vancouver by connecting their needs with the services that their community can offer. Small businesses have a harder time competing in the market to keep their businesses alive, but they have been hit especially hard during this pandemic. This app will help small business owners by providing a directory for all the small businesses that exist in Vancouver (exposure), as well as a platform to connect small business owners who have a need for services (such as the creation of the logo, and translation for flyers) with their volunteers who have the skillset to help them.
 
 As a business owner user, the app will need to store the following data: name of the business, business ID (assigned by the application), location, required services, business description, and status of the store (if they require a service or not). As a community volunteer, the following data are necessary: volunteer name, volunteer ID, general location (city), skills that they can provide, and volunteer description (portfolio or link). Additional functionality that we would like to have given that there is time, is the implementation of a point system so that volunteers can collect non-monetary reimbursement in return for work that is completed which could then be used as a store credit at these businesses.
 
-## Project Task Requirements
+### Built with
+* [React](https://reactjs.org/)
+* [Redux](https://redux.js.org/)
+* [Node.js](https://nodejs.org/)
+* [Express](https://expressjs.com/)
+* [AWS](https://aws.amazon.com/)
```

Comment:
We can specify AWS Amazon S3, and could we add google (maps, OAuth 2.0) here?

Reply:
Sure!